### DISCUSSION TITLE
Deploy wg-easy after setting up cluster

### DISCUSSION THREAD

**Discussion by jeircul** [onedr0p/cluster-template#1400] [General]
Thanks for an amazing cluster-template! 
I have followed the steps in this repo and got a fully working cluster. 
1 master (rpi4) and 3 workers (rpi3). 
What I want to achieve is to deploy wg-easy in my cluster, but I can't wrap my head around how to get it working. 
This example is what I am trying to implement:
https://github.com/wg-easy/wg-easy/wiki/Using-WireGuard-Easy-with-Kubernetes. 
This is my attempt https://github.com/jeircul/home-ops/tree/main/kubernetes/apps/network/wg-easy/app. 
I suspect the persistent storage could be the issue, but I can't find any examples of how to do it.
Any pointers would be greatly appreciated.

**Comment by onedr0p** [comment]
I converted this to a discussion, I'm not exactly sure how to help here. Maybe someone else can.

**Reply by jeircul** [reply]
I managed to make it work, and the problem was lack of knowledge from my side. 
Specifically how the storage claims work and how to use the bjw-s template.
